users/api/http/requests.go

```diff
@@ -31,3 +31,37 @@ func (req viewUserInfoReq) validate() error {
 	}
 	return nil
 }
+
+type passwResetReq struct {
+	Email string
+	Host  string
```

Comment:
If these are json fields, please add json tags.

Reply:
Host is extracted from the referer header